modules/tensor_mechanics/src/materials/ComputeFiniteStrain.C

```diff
@@ -226,8 +255,26 @@ ComputeFiniteStrain::computeQpIncrements(RankTwoTensor & total_strain_increment,
       break;
     }
 
+    case DecompMethod::HughesWinget:
```

Comment:
Can you please document this option in the `md` file.

Reply:
OK I will take a crack at that. Is it worth mentioning this is the method used by Abaqus? Otherwise this third option seems unrelated to the rest of the docs - looking at [ComputeFiniteStrain](https://mooseframework.inl.gov/source/materials/ComputeFiniteStrain.html)

Reply:
Agreed. It seems to me that the current doc is not greatly structured (e.g. volumetric locking correction shouldn't be on the same level as "taylor expansion")... Anyway, it shouldn't be on you to fix that.  I'd create a single section outlining the Hughes-Winget formulation with a reference, I assume [1]. 

> Is it worth mentioning this is the method used by Abaqus? 

Up to you. I am happy with just the reference.

[1]. Hughes, T.J.R. and J. Winget (1980). “Finite rotation effects in numerical integration of rate
constitutive equations arising in large-deformation analysis”. In: International Journal for
Numerical Methods in Engineering 15, pp. 1862–1867.

Reply:
I added a couple of sections to the `ComputeFiniteStrain` documentation.